program/src/instruction.rs

```diff
@@ -101,6 +100,12 @@ pub enum LidoInstruction {
     AddMaintainer,
     RemoveMaintainer,
     MergeStake,
+
+    /// Check if validator increased his commission over maximum allowed
+    /// and deactivate him if he did
+    ///
+    /// Requires no permission
+    CheckMaxCommissionViolation,
```

Comment:
```suggestion
    DeactivateValidatorIfCommissionExceedsMax,
```

How about naming it after what it does?